h/buildext.py

```diff
@@ -225,15 +225,32 @@ def build_chrome(args):
         subprocess_args.append('--debug')
     subprocess.call(subprocess_args)
 
-    # Render the sidebar html.
+
+    # Render the sidebar html
+    base_url = env['request'].route_url('index')
```

Comment:
You define `base_url` but then inconsistently use `args.base` and `base_url` below. This needs to be clear and consistent either way.